tutorials/2_Gazebo_and_ROS.ipynb

```diff
@@ -0,0 +1,1330 @@
+    "      <turnGravityOff>false</turnGravityOff>\n",
+    "    </gazebo>\n",
+    "</robot>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h2>Connect your Robot to ROS</h2>\n",
+    "\n",
+    "<p>Alright, our robot is done nice and has this new car smell, but we can’t do anything with it yet as it has no connection with ROS. In order to add this connection we need to add Gazebo plugins to our model. There are different kinds of plugins:</p>\n",
+    "\n",
+    "<ul>\n",
+    "    <li>World: Dynamic changes to the world, e.g. Physics, like illumination or gravity, inserting models</li>\n",
+    "    <li>Model: Manipulation of models (robots), e.g. move the robots</li>\n",
+    "    <li>Sensor: Feedback from virtual sensor, like camera, laser scanner</li>\n",
+    "    <li>System: Plugins that are loaded by the GUI, like saving images</li>\n",
+    "    </ul>\n",
+    "<p>First of all we’ll use a controller plugin for differential drive robot. The transmission tags in our wheel xacro will be used by this plugin to define how to link the joints to controllers. To activate the plugin, add the following just before closing robot tag.</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "<!-- Differential drive controller  -->\n",
+    "  <gazebo>\n",
+    "    <plugin name=\"differential_drive_controller\" filename=\"libgazebo_ros_diff_drive.so\">\n",
+    "\n",
+    "      <rosDebugLevel>Debug</rosDebugLevel>\n",
+    "      <publishWheelTF>false</publishWheelTF>\n",
+    "      <robotNamespace>/</robotNamespace>\n",
+    "      <publishTf>1</publishTf>\n",
+    "      <publishWheelJointState>false</publishWheelJointState>\n",
+    "      <alwaysOn>true</alwaysOn>\n",
+    "      <updateRate>100.0</updateRate>\n",
+    "      <leftJoint>front_left_wheel_joint</leftJoint>\n",
+    "      <rightJoint>front_right_wheel_joint</rightJoint>\n",
+    "      <wheelSeparation>${2*base_radius}</wheelSeparation>\n",
+    "      <wheelDiameter>${2*wheel_radius}</wheelDiameter>\n",
+    "      <broadcastTF>1</broadcastTF>\n",
+    "      <wheelTorque>30</wheelTorque>\n",
+    "      <wheelAcceleration>1.8</wheelAcceleration>\n",
+    "      <commandTopic>cmd_vel</commandTopic>\n",
+    "      <odometryFrame>odom</odometryFrame> \n",
+    "      <odometryTopic>odom</odometryTopic> \n",
+    "      <robotBaseFrame>base_footprint</robotBaseFrame>\n",
+    "\n",
+    "\n",
+    "    </plugin>\n",
+    "  </gazebo> "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>With this plugin, we will be able to control the robot.</p>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>Now we need to create add our robot model in the launch file we created earlier to spawn the model. Add below in the diff_wheeled_gazebo.launch file </p>\n",
```

Comment:
remove the word 'create'.

Add the lines below to the file: 'diff_wheeled_gazebo.launch'